Dockerfiles/Dockerfile-AlmaLinux8

```diff
@@ -24,6 +24,7 @@ RUN \
 EXPOSE 3002
 
 ENV HOME /root
+ENV DOCKER_DEPLOYMENT=docker
```

Comment:
This should have a `CARTA_` prefix. The point of adding the prefix was to avoid adding a generically named variable into a huge namespace over which we have no control.

Reply:
This should also be a boolean flag with a value of 1 (the current value of `docker` doesn't make sense with the new name).